### PR TITLE
Fix chunk correctness test.

### PR DESCRIPTION
**Pull request by shoaibkamil**
This fixes one of the issues on my system; @slomp PTAL.

### CODE REVIEW

File: test/correctness/chunk.cpp [hunk 1]
```diff
@@ -28,8 +28,22 @@ int main(int argc, char **argv) {
     for (int i = 0; i < 32; i++) {
         for (int j = 0; j < 32; j++) {
```
Comment (abadams) [inline test/correctness/chunk.cpp]: This is one of the most basic tests - it checks if compute_at works. I'd personally prefer switching the test to integers than have it also test D3D math precision.

**Comment by steven-johnson** [comment]
This is now failing in correctness_gpu_mixed_shared_mem_types

**Comment by steven-johnson** [comment]
This PR has been languishing for a while; what's the status?

**Comment by shoaibkamil** [comment]
I think this was pulled into one of @slomp's fixes branches, so ok to close.

**Pull request closed without merging**